applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,148 @@
+# NFT PROVENANCE TRACKER SDK
+
+- **Team Name:** [Metaquity limited](https://www.metaquity.xyz) (Part of Substrate Builders Program)
+- **Payment Address:** Fiat payment (Bank account details will be emailed to grants@web3.foundation by abhishek@metaquity.xyz)
+- **Level:** 2
+
+## Project Overview :page_facing_up:
+
+We propose to build an NFT Provenance Tracker SDK, an on-chain system capable of tracking and displaying the entire lifecycle of NFTs across multiple parachains in the Polkadot ecosystem. The tracker will collect data like NFT creation, past ownership history, price history, metadata, and locking/unlocking events. The data will be made accessible via SDK.
+
+### Project Context
+
+The rise of NFTs has brought a new level of complexity to asset tracking in the blockchain space. The unique and non-interchangeable nature of NFTs, combined with the ability to fractionalize and trade them across multiple parachains, presents a pressing need for a tool that can track their provenance in a trustless and transparent manner.
+
+Our research indicates that, while there are some solutions for tracking NFT provenance, none currently offer cross-parachain functionality combined with an on-chain model for scalability. By developing this tool, we can significantly enhance transparency and trust in the Polkadot NFT space, benefiting individual NFT owners, DEXs, parachain operators, and potential asset buyers.
+
+### Project Details
+
+Architecture:
+
+![](https://i.ibb.co/TkWNWWK/Screenshot-2023-07-03-at-14-13-35.png)
+    
+  A standard asset-tracking SDK will be built with the following layers:
+  - Blockchain layer: Access to the blockchain data source and crawl through the blocks from the genesis block
+  - Middleware layer: Contains the crawlers through which we can crawl the blockchain and collect information about NFTs and write into the database and index it using elastic search.
+  - SDK: This package will have all the necessary functions to fetch details about the NFT asset.
+
+Technology stack: 
+- MongoDB 
+- Typescript 
+- Elasticsearch
+- JavaScript
+- [Rust-web3](https://github.com/tomusdrw/rust-web3)
```

Comment:
What do you use this evm client for? My understanding was that you're focusing on NFTs minted through the Uniques and NFT pallets.

> We will not be focusing currently on the EVM-compatible ones but stick to more Polkadot ecosystem NFT's which are powered mostly by the Uniques Pallet and going forward the NFT's pallet will be used.

Reply:
Thank you for bringing this to our attention. We have made the necessary adjustments.